src/goto-programs/goto_functions_template.h

```diff
@@ -62,6 +62,12 @@ class goto_function_templatet
     parameter_identifiers.clear();
   }
 
+  /// update the function member in each instruction
```

Comment:
Could you elaborate this comment to explain _how_ the instructions_function are being updated.

Reply:
added param description